get.sh

```diff
@@ -39,21 +40,25 @@ getPackage() {
         ;;
         esac
     ;;
+    *_NT*)
+    suffix=".exe"
+    temp=$temp$suffix
+    ;;
     esac
 
-    if [ -e /tmp/faas-cli ]; then
-        rm /tmp/faas-cli
+    if [ -e $temp ]; then
+        rm $temp
     fi
 
     url=https://github.com/openfaas/faas-cli/releases/download/$version/faas-cli$suffix
     echo "Getting package $url"
 
-    curl -sSL $url > /tmp/faas-cli
+    curl -sSL $url > $temp
 
     if [ "$?" = "0" ]; then
         echo "Attemping to move faas-cli to /usr/local/bin"
-        chmod +x /tmp/faas-cli
-        cp /tmp/faas-cli /usr/local/bin/
+        chmod +x $temp
+        cp $temp /usr/local/bin/
```

Comment:
`/usr/local/bin/` is a non-standard location on windows - its not part of the Mingw64 Git Bash install - how do you see this being created on windows?

Reply:
For example, running on Windows 10:
```
$ ./get.sh
which: no faas-cli in (/c/Users/John McCabe/bin:/mingw64/bin:/usr/local/bin:/usr/bin:/bin:/mingw64/bin:/usr/bin:/c/Users/John McCabe/bin:/c/Program Files/Docker/Docker/Resources/bin:/d/Oculus/Support/oculus-runtime:... snip ...)
Getting package https://github.com/openfaas/faas-cli/releases/download/0.4.19/faas-cli.exe
Attemping to move faas-cli to /usr/local/bin
cp: cannot create regular file '/usr/local/bin/': No such file or directory
```

Reply:
@johnmccabe, I hadn't tried it on WSL.  I will give that a go and let you know.  The .exe extension is not needed when just using from Git bash, but is a problem if you try to use it from cmd or Powershell, for instance.  And yes, /usr/local/bin is not a standard location, I just wasn't sure how much to change at once or where it should be placed.

Reply:
Within WSL, uname returns Linux, so it actually installs the Linux binary, and that works correctly within WSL.

Reply:
@johnmccabe, I just tried an install of Git Bash on a new Windows machine, and it appears that /usr/local/bin is part of the default path already, it just hasn't been created yet.  We could do mkdir -p on the script to create if it doesn't exist?  What are your thoughts?

Reply:
Where are bin files stored normally with Git Bash?

Reply:
It appears that some things are in /bin, some are in /mingw64/bin and some
are in /usr/bin.  /usr/local/bin is part of the default search path in Git
Bash, it just isn't created during install.  @johnmccabe, do you concur?

Reply:
A few examples of what I see
```
WESTON@FAAS-WIN MINGW64 ~
$ which curl
/mingw64/bin/curl

WESTON@FAAS-WIN MINGW64 ~
$ which git
/mingw64/bin/git

WESTON@FAAS-WIN MINGW64 ~
$ which ssh
/usr/bin/ssh

WESTON@FAAS-WIN MINGW64 ~
$ which awk
/usr/bin/awk
```